Tests/PointFreeTests/XCTestManifests.swift

```diff
@@ -148,6 +148,18 @@ extension ChangeTests {
     ]
 }
 
+extension DatabaseTests {
+    // DO NOT MODIFY: This is autogenerated, use:
+    //   `swift test --generate-linuxmain`
```

Comment:
I think we can get rid of this with auto-discovery, but can save for another PR later...